Cargo.toml

```diff
@@ -78,7 +78,7 @@ wayland-client = { version = "0.23.0", features = [ "dlopen", "egl", "cursor", "
 mio = "0.6"
 mio-extras = "2.0"
 smithay-client-toolkit = "0.6"
-x11-dl = "2.18.3"
+x11-dl = "2.18.4"
```

Comment:
It's actually version `2.18.5` that's required now.

Reply:
Fixed